sandpack-client/gulpfile.js

```diff
@@ -1,14 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const del = require("del");
 const gulp = require("gulp");
 const removeSourcemaps = require("gulp-remove-sourcemaps");
 
-gulp.task("copy-sandbox", () =>
+const dist = "./sandpack/";
+const paths = process.env.CI
```

Comment:
I wanted to suggest here we drop gulp and find a different way of coping over the bundler, but on the principle: "If it ain't broke, don't fix it", let's leave it like that for now